### PR TITLE
Add id tag attribute and script true/false options

### PR DESCRIPTION
**Pull request by sunchess**
some times you need this two options

### CODE REVIEW

File: lib/recaptcha/client_helper.rb [hunk 1]
```diff
@@ -23,7 +26,10 @@ def recaptcha_tags(options = {})
       data_attributes[:sitekey] = public_key
       data_attributes = data_attributes.map { |k, v| %(data-#{k.to_s.tr('_', '-')}="#{v}") }.join(" ")
 
-      html = %(<script src="#{script_url}" async defer></script>\n)
+      data_attributes << %( id="#{options[:id]}") if options[:id]
+
+      html = ""
+      html << %(<script src="#{script_url}" async defer></script>\n) if options[:script]
```
Comment (grosser) [inline lib/recaptcha/client_helper.rb]: `if options.fetch(:script, true)`

File: test/client_helper_test.rb [hunk 2]
```diff
@@ -32,4 +32,18 @@
       recaptcha_tags
     end
   end
+
+  it "including id in tag" do
```
Comment (grosser) [inline test/client_helper_test.rb]: `it including id in tag` <- what kind of english is that :)

File: lib/recaptcha/client_helper.rb [hunk 3]
```diff
@@ -23,7 +26,10 @@ def recaptcha_tags(options = {})
       data_attributes[:sitekey] = public_key
       data_attributes = data_attributes.map { |k, v| %(data-#{k.to_s.tr('_', '-')}="#{v}") }.join(" ")
 
-      html = %(<script src="#{script_url}" async defer></script>\n)
+      data_attributes << %( id="#{options[:id]}") if options[:id]
```
Comment (grosser) [inline lib/recaptcha/client_helper.rb]: it's not a data_attribute, so this looks a bit strange ... add it straight into the html below ?

**Comment by sunchess** [comment]
ok, fixed according notes :)

**Comment by grosser** [comment]
3.3.0 FYI cleanup https://github.com/ambethia/recaptcha/commit/5f01ef4ee695b7faca239863e2235b6ee4e786c6

**Pull request merged**